doc/source/whatsnew/v0.18.0.txt

```diff
@@ -440,6 +439,7 @@ Bug Fixes
 - Bug in consistency of passing nested dicts to ``.groupby(...).agg(...)`` (:issue:`9052`)
 - Accept unicode in ``Timedelta`` constructor (:issue:`11995`)
 
+- Bug in value label reading for ``StataReader`` when reading incrementally (:issue:`12014`)
```

Comment:
this is also a perf enhancement, yes? pls add a line in Performance (use this PR number for it)